app/presenters/guide_presenter.rb

```diff
@@ -41,6 +41,28 @@ def parts
     end
   end
 
+  def hide_chapter_navigation?
+    hide_navigation_elements = content_item.parsed_content["details"]["hide_chapter_navigation"]
+    step_navs = content_item.parsed_content["links"]["part_of_step_navs"]
+
+    part_of_step_navs = false
+    part_of_step_navs = step_navs.length if step_navs
```

Comment:
I'd extract these three lines out to a method, `part_of_step_navs?`

I think it could also be reduced to one line checking whether `content_item.parsed_content["links"]["part_of_step_navs"]` exists and the size is greater than 1.